rg3d-ui/src/text_box.rs

```diff
@@ -906,6 +908,16 @@ impl<M: MessageData, C: Control<M, C>> TextBoxBuilder<M, C> {
         self
     }
 
+    pub fn with_mask_char(mut self, mask_char: char) -> Self {
```

Comment:
I think one method that accepts `Option<char>` should be enough. `with_no_mask_char` is a bit confusing because you have no mask char by default.